modules/publicize/publicize.php

```diff
@@ -119,7 +119,7 @@ function __construct() {
 
 		add_action( 'init', array( $this, 'add_post_type_support' ) );
 		add_action( 'init', array( $this, 'register_post_meta' ), 20 );
-		add_action( 'init', array( $this, 'register_gutenberg_extension' ), 30 );
+		add_action( 'rest_api_init', array( $this, 'register_gutenberg_extension' ), 30 );
```

Comment:
Do we need this `30` priority?

Reply:
(Since I've reverted `rest_api_init` to the previous `init`, I guess I'm also leaving the `30` untouched so the line is entirely unchanged.)